src/core/constants.js

```diff
@@ -22,8 +22,18 @@ export const VERSION =
  */
 export const P2D = 'p2d';
 /**
- * One of the two render modes in p5.js: P2D (default renderer) and WEBGL
- * Enables 3D render by introducing the third dimension: Z
+ * One of the two render modes in p5.js, most known for 3D rendering.
+ *
+ * Some differences and new features to expect compared to the default render mode <a href="/#/p5/P2D">P2D</a>:
```

Comment:
Can we simplify the language here:
`WEBGL differs from the default P2D renderer in the following ways:`